pandasai/llm/google_gemini.py

```diff
@@ -87,12 +87,8 @@ def _generate_text(self, prompt: str, memory: Optional[Memory] = None) -> str:
         updated_prompt = self.prepend_system_prompt(prompt, memory)
 
         self.last_prompt = updated_prompt
-        completion = self.google.GoogleGemini.generate_content(
+        completion = self.google_gemini.generate_content(
             contents=prompt,
-            temperature=self.temperature,
-            top_p=self.top_p,
-            top_k=self.top_k,
-            max_output_tokens=self.max_output_tokens,
```

Comment:
Why did you remove these params?

Reply:
I got this error:
File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/pandasai/llm/google_gemini.py", line 91, in _generate_text
    completion = self.google_gemini.generate_content(
  File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/google/generativeai/generative_models.py", line 234, in generate_content
    request = self._prepare_request(
  File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/google/generativeai/generative_models.py", line 215, in _prepare_request
    return glm.GenerateContentRequest(
  File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/proto/message.py", line 576, in __init__
    raise ValueError(
ValueError: Unknown field for GenerateContentRequest: temperature


but I think we can solve it without removing params, will do it on the next commit

Reply:
@traien thanks a lot, merging!